### PR TITLE
Add support of pool_autostart

### PR DESCRIPTION
**Pull request by hahakiki2010**
libvirt_storage: Add support of pool_autostart, to get
pool autostart status

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

File: virttest/libvirt_storage.py [hunk 1]
```diff
@@ -184,6 +184,24 @@ def pool_state(self, name):
...
+            pools = self.list_pools()
+        except process.CmdError:
+            return None
+        if self.pool_exists(name):
+            details_dict = pools[name]
+            try:
+                return details_dict['Autostart']
+            except KeyError:
+                pass
+        return None
```
Comment (balamuruhans) [inline virttest/libvirt_storage.py]:
```
try:
    return self.list_pools()[name]["Autostart"]
except process.CmdError, KeyError:
    return None
```

Comment (hahakiki2010) [reply virttest/libvirt_storage.py]: Is it for more simple or something else?

Comment (balamuruhans) [reply virttest/libvirt_storage.py]: `pool_exists` also calls `list_pools()`, this redundant calls can be avoided.

Comment (hahakiki2010) [reply virttest/libvirt_storage.py]: Agreed and updated code. beside, pool_state is updated in this way also.

**Pull request merged**